legend-engine-executionPlan-execution-store-relational-connection-tests-api/src/main/java/org/finos/legend/engine/plan/execution/stores/relational/connection/tests/api/ConnectionTestManager.java

```diff
@@ -0,0 +1,29 @@
+// Copyright 2020 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.finos.legend.engine.plan.execution.stores.relational.connection.tests.api;
+
+import org.finos.legend.engine.protocol.pure.v1.model.packageableElement.store.relational.connection.DatabaseType;
+
+public interface ConnectionTestManager
+{
+    DatabaseType getDatabaseType();
+
+    String getPureTestCollectionPath();
+
+    default DynamicTestConnection getDynamicTestConnection()
```

Comment:
Why do we need a no-op impl that returns null ?  The whole point of a test manager is to provide test connections.
Implementers of ConnectionTestManager should implement getDynamicTestConnection (or they can chose to return null in the implementation.

Reply:
Implementers can signal that they can't implement getDynamicTestConnection by skipping it and relying on default. This default null impl also documents that null is a valid value used to signal that dynamic connections are not supported.

Reply:
removed this interface